### PR TITLE
fix(widget): add base url to reverse and autocomplete fetch path

### PR DESCRIPTION
**Pull request by mortennordseth**
This adds the base URL config to the autocomplete and reverse fetch.

### CODE REVIEW

File: src/widget/widget.ts [hunk 1]
```diff
@@ -935,9 +935,9 @@ async function autocomplete(q: string) {
   return data;
 }
 
-export async function reverse(coords: GeolocationCoordinates) {
+export async function reverse(urlBase: string, coords: GeolocationCoordinates) {
   const result = await fetch(
-    `/api/departures/reverse?lat=${coords.latitude}&lon=${coords.longitude}`,
+    `${urlBase}api/departures/reverse?lat=${coords.latitude}&lon=${coords.longitude}`,
```
Comment (mikaelbr) [inline src/widget/widget.ts:940]: This now requires leading `/` in the urlBase, but I think icons does not require it. So should either: always ensure leading slash or be consistant. Think ensuring leading slash is the best here

Comment (adriansberg) [reply src/widget/widget.ts:940]: Isn't that ensured here: https://github.com/AtB-AS/planner-web/blob/0e7fbcde1f51e2381684192a67a0a2d00a895996/src/widget/widget.ts#L29 ?

Comment (mortennordseth) [reply src/widget/widget.ts:940]: This should be handled in the `createSettingsConstants` 🤔 

```
if (!urlBase.endsWith('/')) {
    urlBase += '/';
  }
```

Comment (mikaelbr) [reply src/widget/widget.ts:940]: Dang it. Too smart for my own good

**Review by mikaelbr — APPROVED** [review]
Looks good!

**Pull request merged**